openshift_scalability/scripts/ptibenchmark.sh

```diff
@@ -0,0 +1,17 @@
+# The following script will execture A B Benchmark for pti(spectre).
+
+#TODO: Abstract this script to have an A B node vertical concept, but do some changes in between the runs.
+
+#Optimize the register script to run this command there.
+for h in `oc get no |grep -v NAME|awk '{print $1}'`; do ssh $h "echo "export LC_CTYPE=en_US.UTF-8 >> ~/.bashrc; export LC_ALL=en_US.UTF-8 >> ~/.bashrc; pbench-register-tool-set; done
+
+#PTI Enabled
+for h in `oc get no |grep -v NAME|awk '{print $1}'`; do ssh $h "echo 1 > /sys/kernel/debug/x86/pti_enabled"; done 
+nohup $(pbench-user-benchmark ~/svt/openshift_scalability/./nodeVertical.sh ptibench python > /tmp/n.log 2>&1)  &
+pbench-move-results --prefix=pti_enabled
+
+#PTI Disabled
+for h in `oc get no |grep -v NAME|awk '{print $1}'`; do ssh $h "echo 0 > /sys/kernel/debug/x86/pti_disabled"; done 
```

Comment:
kernel parameter is pti_enabled, not pti_disabled

Reply:
right, this will make sure \ force we will test both cases